lib/bundle/commands/list.rb

```diff
@@ -7,7 +7,7 @@ module List
 
       def run(global: false, file: nil, all: false, casks: false, taps: false, mas: false, whalebrew: false,
               vscode: false, brews: false)
-        parsed_entries = Bundle::Dsl.new(Brewfile.read(global:, file:)).entries
+        parsed_entries = Brewfile.read(global:, file:).entries
```

Comment:
This affords that `Brewfile.read` should be renamed one of:

* `Brewfile.parse` 
* `Brewfile.read_and_parse`
* `Brewfile.slurp`